charts/ocis/docs/values.adoc.yaml

```diff
@@ -320,8 +338,24 @@ features:
           # -- Disables SSL certificate checking for connections to the office suites http api.
           # Not recommended for production installations.
           insecure: false
-          # -- Disables Chat functionality of OnlyOffice
-          disableChat: false
```

Comment:
disable chat functionality will be possible with the next rolling release: including https://github.com/owncloud/ocis/pull/9580

Reply:
tracked in ticket https://github.com/owncloud/ocis-charts/issues/640